entrypoint.sh

```diff
@@ -24,17 +24,19 @@ else
     SQL_TYPE=$(echo "$ARCHIVISTA_SQL_STORE_BACKEND" | tr '[:lower:]' '[:upper:]')
   fi
   case $SQL_TYPE in
-  MYSQL)
+  MYSQL*)
     if [[ -z $ARCHIVISTA_SQL_STORE_CONNECTION_STRING ]]; then
       ARCHIVISTA_SQL_STORE_CONNECTION_STRING="root:example@db/testify"
     fi
     echo "Running migrations for MySQL"
-    atlas migrate apply --dir "file:///archivista/migrations/mysql" --url "mysql://$ARCHIVISTA_SQL_STORE_CONNECTION_STRING"
+    DB_URL=$(/bin/archivistactl iam "${SQL_TYPE}" "mysql://${ARCHIVISTA_SQL_STORE_CONNECTION_STRING}")
```

Comment:
does this change to the entrypoint always assume this is being run in an AWS environment with IAM?

If so, this could break existing instances after release. 
We may need to invoke the `iam` command conditionally instead

Reply:
ahh, I see now. The `iam` command will not convert the connection string if the connection string is now for AWS. 

I think that will work, but it is a little strange to see `iam` used in places, regardless if iam is present.